test/cases/active_job_serializer_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'abstract_unit'
+
+require 'fixtures/project'
+require 'fixtures/person'
+require 'active_job'
+require 'active_job/arguments'
+
+class ActiveJobSerializerTest < ActiveSupport::TestCase
+  require 'active_resource/active_job_serializer'
+
+  setup do
+    @klass = ActiveResource::ActiveJobSerializer
+    @person = Person.new({ id: 2, name: 'David' })
+    @person.persisted = true
+
+    @project = Project.new({ id: 1, name: 'Ruby on Rails' })
+    @project.prefix_options[:person_id] = @person.id
```

Comment:
I don't like instance variables in setup blocks. They remove duplication, but they also introduce the mystery guest anti-pattern when reading the test. Better to inline each one of those variables in the tests that uses them.